Header header
bool isConnected
bool takeControl
vector3D linear
vector3D angular